### PR TITLE
chore: add new config options into packages config with default values

### PR DESCRIPTION
**Pull request by vandot**
Added all new config options into packages config files with default values and descriptions

---
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1794)

### CODE REVIEW

**Review by Eknir — APPROVED** [review]
LGTM!

File: packaging/docker/env [hunk 1]
```diff
@@ -61,6 +65,8 @@ BEE_CLEF_SIGNER_ENABLE=true
 # BEE_PAYMENT_THRESHOLD=10000000000000
 ## excess debt above payment threshold in BZZ where you disconnect from your peer (default 10000000000000)
 # BEE_PAYMENT_TOLERANCE=10000000000000
+## postage stamp contract address
+# BEE_POSTAGE_STAMP_ADDRESS=
```
Comment (acud) [inline packaging/docker/env:69]: @ralph-pichler is this needed here? maybe a default value?

File: packaging/docker/env [hunk 2]
```diff
@@ -71,14 +77,18 @@ BEE_CLEF_SIGNER_ENABLE=true
 # BEE_SWAP_ENDPOINT=ws://localhost:8546
 ## swap factory address
 # BEE_SWAP_FACTORY_ADDRESS=
-## initial deposit if deploying a new chequebook (default 100000000)
-# BEE_SWAP_INITIAL_DEPOSIT=100000000
+## legacy swap factory addresses
+# BEE_SWAP_LEGACY_FACTORY_ADDRESSES=
```
Comment (acud) [inline packaging/docker/env:81]: ditto

**Review by acud — APPROVED** [review]
minor comments. input from @ralph-pichler welcome

File: packaging/docker/env (hunk 1 above)
Comment (ralph-pichler) [reply packaging/docker/env:69]: no default value needed here. the goerli addresses will be detected automatically. this flag and the other one only exists for test blockchain deployments and unsupported networks.

**Pull request merged**